### PR TITLE
Snippets config: Remove CPP triggers; Update to .NET 5 RC 1

### PR DESCRIPTION
**Pull request by adegeo**

### CODE REVIEW

File: .github/workflows/build-validation.yml [hunk 1]
```diff
@@ -9,19 +9,16 @@ on:
       - "**.cs"
       - "**.vb"
       - "**.fs"
-      - "**.cpp"
-      - "**.h"
       - "**.xaml"
       - "**.razor"
       - "**.cshtml"
       - "**.vbhtml"
-      - "**.*proj"
```
Comment (BillWagner) [inline .github/workflows/build-validation.yml:18]: We should keep the `csproj`, `vsproj`, and `vbproj` so that we build projects when dependencies change.

Comment (adegeo) [reply .github/workflows/build-validation.yml:18]: DUH on me.

File: .github/workflows/build-validation.yml [hunk 2]
```diff
@@ -9,19 +9,19 @@ on:
...
+      - "**.csproj"
+      - "**.vbproj"
+      - "**.fsproj"
       - "**global.json"
       - "**snippets.5000.json"
     branches: [ master ]
 
 env:
-  DOTNET_INSTALLER_CHANNEL: 'release/5.0.1xx-preview8'
+  DOTNET_INSTALLER_CHANNEL: 'release/5.0.1xx-preview7'
```
Comment (pkulikov) [inline .github/workflows/build-validation.yml:24]: RC1 is released; would roll forward work?

**Review by BillWagner — APPROVED** [review]
This LGTM @adegeo 

I wonder about @pkulikov 's comment as well. Is it right just to move to RC 1?

**Comment by adegeo** [comment]
I believe this will work. I'll test here and then do the edits in the other repos

**Pull request merged**